.github/workflows/preview.yml

```
@@ -0,0 +1,30 @@
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+
+      - name: Install deps
+        run: pip install -r requirements.txt
+
+      - name: Build site
+        run: python build.py   # script build ra HTML
+
+      - name: Deploy preview
+        uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.11)

[error] the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)